src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do list</title>
+    <script src="https://kit.fontawesome.com/9c25278351.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <section>
+        <div class="container">
+            <header>
+                <h1>To Do Lists</h1>
+            </header>
+            <div class="todolist-container">
+                <h2>Today's To Do<i class="fas fa-sync icon1"></i></h2>
+                <form id="taskForm" action="POST" class="input-icon">
+                    <input class="todo-input" type="text" name="New Task" id="newTask" placeholder="Add to your list..." required>
```

Comment:
Kindly add a label to the form input elements to ensure you comply with HTML best practices, i.e accessibility. 😄